TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/Sources/CCOLGeneratorGenerateOvC.cs

```diff
@@ -680,6 +688,7 @@ private string GenerateOvCRijTijdScenario(ControllerModel c)
                                     {
                                         sb.AppendLine($"NG, {_dpf}{d.Naam}, NG);");
                                     }
+                                    break;
```

Comment:
waarom een break? dan zal een tweede lus of lussenpaar buiten beschouwing worden gelaten.